apps/datacalc.py

```diff
@@ -138,3 +143,4 @@ def calculateSum(self):
             result += value
         else:
             self.viewerFrame.numberLabel.setText(f"sum: {result}")
+            self.broadcastRequested.emit("logbus", f"Calculate the sum: {result}.")
```

Comment:
Using an imperative for logging messages looks a little awkward.
It is also not consistent with the database message above.
How about "Calculated the sum:", or just "Sum:"?

Reply:
You are very meticulous! I will use "Sum:".